components/canvas/webgl_thread.rs

```diff
@@ -97,11 +98,20 @@ impl<VR: WebVRRenderHandler + 'static, OB: WebGLThreadObserver> WebGLThread<VR,
                                     .expect("WebGLContext not found");
                     let glsl_version = Self::get_glsl_version(ctx);
 
+                    let mut max_combined_texture_image_units = [0];
+                    unsafe {
+                        ctx.gl().get_integer_v(
```

Comment:
This duplicates https://github.com/servo/rust-offscreen-rendering-context/blob/master/src/gl_limits.rs#L86.